### ISSUE TITLE
Provide a pre-setup "ethers" that gives you a provider pointing to local validator

### ISSUE THREAD

**Issue by sanderpick** [tablelandnetwork/local-tableland#75]

**Comment by joewagner** [comment]
@sanderpick we have this method https://github.com/tablelandnetwork/local-tableland/blob/main/src/util.ts#L242
which lets users of this lib do things like:
```
import { connect } from "@tableland/sdk";
import { getAccounts } from "@tableland/local";

const localAccounts = getAccounts();
const tableland = connect({
  signer: localAccounts[7]
});
```

Does that look like it meets the needs for this issue?

**Comment by carsonfarmer** [comment]
Looks like this is fixed with #103
